scalableminds/webknossos
# Task Assignment via openInstances Column

yet another attempt at #2482

### Steps to test:
- create tasks, request them as different users
- delete / update annotations, add instances via edit task (beware #2596) and via bulk increase in project view
- number of open instances should stay correct
- generally attempt to break this consistency

### TODO
- [x] add db evolution
- [x] stress-test how this handles parallel requests [edit: faster, retries a lot, no misbehavior locally]
- [ ] clean up negative open instances in already-existing tasks

------
